### ISSUE TITLE
Visibility filter based on Mime Type for "Open With" extension point

### ISSUE THREAD

**Issue by angelborroy-ks** [Alfresco/alfresco-content-app#689]
## I'm submitting a...
<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[X] Feature request
[ ] Documentation issue or request
[ ] Support request
</code></pre>

## Current behavior
Extension point `features.viewer.openWith` has no property to control visibility by mimetype. 

## Expected behavior
```
"features": {
    "viewer": {
        "openWith": [
            ...
            "fileExtension": "docx",
            ....
        ]
    }
}
```

## What is the motivation / use case for changing the behavior?
I was thinking on adding an "Open with Office" extension when I realised that, as no every mimetype should be affected by this extension.

## Environment

<pre><code>
ACA version: 1.4.0

Browser:
- [X] Chrome version XX
- [] Firefox version XX
- [ ] Safari version XX
- [ ] Edge version XX
- [ ] IE11
 
For Tooling issues:
- Node version: 10.9.0
- Platform:  OS X Mojave

Others:
</code></pre>

**Comment by angelborroy-ks** [comment]
It should be also nice to switch the viewer but it's not required for AOS integration. 

What is also required is SSL configuration, what (afaik) it's not documented for ACA. Should I raise another issue for that?

**Comment by DenysVuika** [comment]
Fixed in upcoming 1.7.0. We managed to solve the problem with the dedicated evaluator that toggles visibility based on the content.
